src/main/scala/com/campudus/tableaux/controller/MediaController.scala

```diff
@@ -16,22 +16,27 @@ import scala.reflect.io.Path
 import scala.util.{Failure, Success, Try}
 
 object MediaController {
+
   /**
     * Alias for None which represents
     * the root folder (which doesn't
     * really exist)
     */
   val ROOT_FOLDER = Folder(0, "root", "", Seq.empty[FolderId], None, None)
 
-  def apply(config: TableauxConfig, folderModel: FolderModel, fileModel: FileModel, attachmentModel: AttachmentModel): MediaController = {
+  def apply(config: TableauxConfig,
+            folderModel: FolderModel,
+            fileModel: FileModel,
+            attachmentModel: AttachmentModel): MediaController = {
     new MediaController(config, folderModel, fileModel, attachmentModel)
   }
 }
 
 class MediaController(override val config: TableauxConfig,
                       override protected val repository: FolderModel,
                       protected val fileModel: FileModel,
-                      protected val attachmentModel: AttachmentModel) extends Controller[FolderModel] {
+                      protected val attachmentModel: AttachmentModel)
+    extends Controller[FolderModel] {
```

Comment:
`extends ....` in next line is better. The params for the class in just two spaces would be nicer IMO (like I said for methods definition and calling).